document/core/text/modules.rst

```diff
@@ -471,14 +471,12 @@ Element Segments
 ~~~~~~~~~~~~~~~~
 
 Element segments allow for an optional :ref:`table index <text-tableidx>` to identify the table to initialize.
-When omitted, :math:`\T{0}` is assumed.
 
 .. math::
    \begin{array}{llclll}
    \production{element segment} & \Telem_I &::=&
-     \text{(}~\text{elem}~~(x{:}\Ttableidx_I)^?~~\text{(}~\text{offset}~~e{:}\Texpr_I~\text{)}~~y^\ast{:}\Tvec(\Tfuncidx_I)~\text{)} \\ &&& \qquad
-       \Rightarrow\quad \{ \ETABLE~x', \EOFFSET~e, \EINIT~y^\ast \} \\
-       &&& \qquad\qquad\qquad (\iff x' = x^? \neq \epsilon \vee x' = 0) \\
```

Comment:
Might want to uplift this change to the v.1 spec as an editorial fix.  The new way is much cleaner and I'm even sure if this expression parses...